p2panda-rs/src/lib.rs

```diff
@@ -92,6 +92,16 @@ pub trait Validate {
     fn validate(&self) -> Result<(), Self::Error>;
 }
 
+/// Trait used by p2panda structs for human-facing functionality, like better readability.
```

Comment:
I like the trait name :D We could use the trait's docstring to differentiate it more explicitly from `Display` and `Debug`. `Display` in particular also claims to be made for human-facing display but we have found out that it's not very well suited for that.

Reply:
I really like the name, and also in general how we're using a trait here to better explain and contain a functionality. Along with understanding how we use `Validate` better I think this may be opening the door to some nice p2panda trait inventions ;-p